MANIFEST.in

```diff
@@ -1 +1 @@
-include README.md requirements.txt LICENSE
```

Comment:
requirements should be a dev thing now, not a prod thing so I popped it outta here